give/src/components/Button/Button.js

```diff
@@ -1,12 +1,19 @@
+// Based on v2
+// https://www.aemcomponents.dev/content/core-components-examples/library/core-content/button.html
+// https://github.com/adobe/aem-core-wcm-components/tree/main/content/src/content/jcr_root/apps/core/wcm/components/button/v2/button
+
+import { ComponentWrapper } from "../shared";
 import "./Button.css";
 
 const ButtonLink = ({ href, children }) => {
-  return href !== "" ? (
+  return href ? (
```

Comment:
If a button AEM component is provided a valid URL, it will render as an `<a>`. Otherwise, it'll render as a `<button>`. Compare the standard and linked versions' markup from the [documentation](https://www.aemcomponents.dev/content/core-components-examples/library/core-content/button.html).